curvefs/src/metaserver/storage/memory_storage.cpp

```diff
@@ -296,28 +296,19 @@ Status MemoryStorage::Rollback()  {
     return Status::OK();
 }
 
-bool MemoryStorage::GetStatistics(StorageStatistics* statistics) {
-    statistics->maxMemoryQuotaBytes = options_.maxMemoryQuotaBytes;
-    statistics->maxDiskQuotaBytes = options_.maxDiskQuotaBytes;
-
-    // memory usage bytes
-    if (!GetProcMemory(&statistics->memoryUsageBytes)) {
-        return false;
-    }
-
-    // disk usage bytes
-    uint64_t total, available;
-    if (!GetFileSystemSpaces(options_.dataDir, &total, &available)) {
-        LOG(ERROR) << "Get filesystem space failed.";
-        return false;
-    }
-    statistics->diskUsageBytes = total - available;
+StorageOptions MemoryStorage::GetStorageOptions() const {
+    return options_;
+}
 
-    return true;
+bool MemoryStorage::Checkpoint(const std::string& dir,
```

Comment:
Note that later versions do not support memorykv in commit message.

Reply:
fix